README.md

```diff
@@ -133,6 +132,20 @@ Your last Laptop run will be saved to `~/laptop.log`. Read through it to see if
 you can debug the issue yourself. If not, copy and paste the entire log into a
 [new GitHub Issue](https://github.com/18F/laptop/issues/new) for us.
 
+#### Git Seekrets False Positives
+
+Sometimes the `git-seekrets` rules may indicate a false positive and matching
+things at aren't actually secrets. This can be solved in a number of ways, but
+the root cause is that the regular expressions used to `match` and `unmatch` for
+the specific rule that is being triggered and should be evaluated.
```

Comment:
Looks good overall. This paragraph has a few typos and can be simplified I think. What do you think of this change?
```
Sometimes the `git-seekrets` rules may indicate a false positive and match 
things that aren't actually secrets. This can happen if the regular 
expressions used to `match` and `unmatch` are too strict.
```